azurerm/resource_arm_batch_pool_test.go

```diff
@@ -0,0 +1,319 @@
+		if err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return err
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testaccAzureRMBatchPool_fixedScale_complete(rInt int, rString string, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+	name     = "testaccbatch%d"
+	location = "%s"
+}
+
+resource "azurerm_storage_account" "test" {
+	name                     = "testaccsa%s"
+	resource_group_name      = "${azurerm_resource_group.test.name}"
+	location                 = "${azurerm_resource_group.test.location}"
+	account_tier             = "Standard"
+	account_replication_type = "LRS"
+}
+
+resource "azurerm_batch_account" "test" {
+	name                 = "testaccbatch%s"
+	resource_group_name  = "${azurerm_resource_group.test.name}"
+	location             = "${azurerm_resource_group.test.location}"
+	pool_allocation_mode = "BatchService"
+	storage_account_id   = "${azurerm_storage_account.test.id}"
+
+	tags {
+		env = "test"
+	}
+}
+
+resource "azurerm_batch_pool" "test" {
+	name                   = "testaccpool%s"
+	resource_group_name    = "${azurerm_resource_group.test.name}"
+	account_name 		   = "${azurerm_batch_account.test.name}"
+	display_name		   = "Test Acc Pool"
+	vm_size				   = "Standard_A1"
+	scale_mode			   = "Fixed"
+	target_dedicated_nodes = 2
+	node_agent_sku_id	= "batch.node.ubuntu 16.04"
+
+	storage_image_reference {
+        publisher = "Canonical"
+        offer     = "UbuntuServer"
+        sku       = "16.04.0-LTS"
+        version   = "latest"
+	}
+  }
+`, rInt, location, rString, rString, rString)
+}
+
+func testaccAzureRMBatchPool_autoScale_complete(rInt int, rString string, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+	name     = "testaccbatch%d"
+	location = "%s"
+}
+
+resource "azurerm_storage_account" "test" {
+	name                     = "testaccsa%s"
+	resource_group_name      = "${azurerm_resource_group.test.name}"
+	location                 = "${azurerm_resource_group.test.location}"
+	account_tier             = "Standard"
+	account_replication_type = "LRS"
+}
+
+resource "azurerm_batch_account" "test" {
+	name                 = "testaccbatch%s"
+	resource_group_name  = "${azurerm_resource_group.test.name}"
+	location             = "${azurerm_resource_group.test.location}"
+	pool_allocation_mode = "BatchService"
+	storage_account_id   = "${azurerm_storage_account.test.id}"
+
+	tags {
+		env = "test"
+	}
+}
+
+resource "azurerm_batch_pool" "test" {
+	name                   		  = "testaccpool%s"
+	resource_group_name           = "${azurerm_resource_group.test.name}"
+	account_name 		   		  = "${azurerm_batch_account.test.name}"
+	display_name		   		  = "Test Acc Pool"
+	vm_size				   		  = "Standard_A1"
+	scale_mode			   		  = "Auto"
+	autoscale_evaluation_interval = "PT15M"
+	autoscale_formula			  = <<EOF
+	startingNumberOfVMs = 1;
```

Comment:
Could we fix the assignment alignment here?

Reply:
Done.